pkgs/development/libraries/plasma-wayland-protocols/default.nix

```diff
@@ -5,11 +5,11 @@
 
 mkDerivation rec {
   pname = "plasma-wayland-protocols";
-  version = "1.3.0";
+  version = "1.4.0";
```

Comment:
Why is this not part of srcs?

Reply:
I'm not sure, maybe this library is not considered part of the official kde-frameworks distribution/release, and/or has different release schedule.